examples/pie_and_polar_charts/pie_and_donut_labels.py

```diff
@@ -43,7 +43,7 @@
 
 
 def func(pct, allvals):
-    absolute = int(round(pct/100.*np.sum(allvals)))
+    absolute = round(pct / 100 * np.sum(allvals))
```

Comment:
These also need `round(float(...))` or it will fail with older numpys (right now we only build docs with numpy 1.19 so this doesn't fail CI, but I'd say we should try to make examples support the same range of numpys as the library).

Also I'd prefer `int(round(...))` rather than `round(float(...))` as the latter could in theory mishandle strings ("1.2"), too (obviously, depending on the context, as string input may be impossible).

Reply:
Docs now build correctly with old NumPy, so if this were rebased, it would pick that up and fail if it were wrong.